app/views/student_mailer/disband_notification.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
I know it's annoying, and we'll definitely want to refactor this (opening up an issue now), but can we include both HTML and text versions of all e-mails?